setup.py

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/python
```

Comment:
I'm not sure what purpose it serves (do you execute setup.py as a script - not as `python setup.py install`)? But it does have executable bit set, and actually didn't solve the problem I uncovered with the wrong `python` executing it - so I put it back in. Ended up moving to `pip install` instead, since that retains metadata for people to uninstall (if installed from source, which we commonly recommend).